src/main/java/net/neoforged/neoforge/common/world/ModifiableBiomeInfo.java

```diff
@@ -80,7 +79,7 @@ public BiomeInfo getModifiedBiomeInfo() {
     @ApiStatus.Internal
     public boolean applyBiomeModifiers(final Holder<Biome> biome, final List<BiomeModifier> biomeModifiers, RegistryAccess registryAccess) {
         if (this.modifiedBiomeInfo != null)
-            throw new IllegalStateException(String.format(Locale.ENGLISH, "Biome %s already modified", biome));
+            return true;
```

Comment:
Adjust the javadoc of this method to match how this method now operates. (It no longer throws an exception, and the biome is only modified on the first invocation of this method.)